codeserver/ubi9-python-3.9/Dockerfile

```diff
@@ -39,6 +39,24 @@ RUN yum install -y "https://github.com/coder/code-server/releases/download/${COD
 RUN chmod -R g+w /opt/app-root/lib/python3.9/site-packages && \
     fix-permissions /opt/app-root -P
 
+# Create a temporary directory for extensions
+WORKDIR /opt/app-root/extensions-temp
+
+COPY --chown=1001:0 utils/*.vsix /opt/vsix/
```

Comment:
any specific reason to copy it to a separate file, than just copying everything from `utils` to `utils` ?